lib/options/dcu.robot

```diff
@@ -55,6 +55,10 @@ Get UEFI Option
     ...    shell=True
     RETURN    ${out.stdout}
 
+Reset UEFI Options to Defaults
```

Comment:
Universal KW to reset the settings via ssh. Maybe we should flash only the section containing the settings here?